app/assets/javascripts/govuk_publishing_components/load-analytics.js

```diff
@@ -71,10 +73,9 @@ window.GOVUK.loadAnalytics = {
     }
   },
 
-  loadGa4: function (currentDomain) {
+  loadGa4Vars: function (currentDomain) {
     currentDomain = currentDomain || window.location.hostname
     var environment = ''
-
     // Categorise current environment
     if (this.arrayContains(currentDomain, this.productionDomains)) {
```

Comment:
Nitpick: this code isn't part of the PR but I was wondering if there'd be any mileage in moving these parameters into an object so there's no reliance on passing them in the correct order? Happy to leave this as it is though as there are only 2 parameters, was just a thought that occurred to me.